packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapPolylineController.m

```diff
@@ -12,6 +12,15 @@ @interface FLTGoogleMapPolylineController ()
 
 @end
 
+
+/**
+ * Returns dict[key], or nil if dict[key] is NSNull.
+ */
+id GetValueOrNilFromDict(NSDictionary * dict, NSString *key) {
+    id value = dict[key];
+    return value == [NSNull null] ? nil : value;
+}
```

Comment:
It's fine to leave this in the file that needs it, although it should be declared `static`.

(It will go way soon, when we convert to Pigeon, so making a new file will just add extra work to undo later.)